resources/precompiled-js/application.js

```diff
@@ -196,14 +192,28 @@ $(document).ready(function () {
       //show the fa questions when checked
       $("#fa-questions").fadeIn();
       $("#fa-questions :input").prop("required", true);
-      $("#how_complete_course").prop("required", false); //hide standard questions
-
-      $("#how_to_enroll").prop("checked", false);
-      $("#standard-questions").fadeOut();
-      $("#standard-questions :input").prop("required", false);
+      $("#how_complete_course").prop("required", false);
     } else {
       $("#fa-questions").fadeOut();
       $("#fa-questions :input").prop("required", false);
     }
+  });
+  $('input:radio[id="belongs_to_flood_area"]').change(function () {
+    if ($(this).is(":checked") && $(this).val() == "1") {
+      $("#flood_area_questions").fadeIn();
+      $("#flood_area_questions :input").prop("required", true);
+    } else {
+      $("#flood_area_questions").fadeOut();
+      $("#flood_area_questions :input").prop("required", false);
+    }
+  });
+  $('input:radio[id="belongs_to_flood_area_no"]').change(function () {
+    if ($(this).is(":checked") && $(this).val() == "0") {
+      $("#flood_area_questions").fadeOut();
+      $("#flood_area_questions :input").prop("required", false);
+    } else {
+      $("#flood_area_questions").fadeIn();
+      $("#flood_area_questions :input").prop("required", true);
+    }
   }); // form validation
 });
```

Comment:
Also can we vertically align the following text?
<img width="1245" alt="image" src="https://user-images.githubusercontent.com/34311857/194852480-92deac09-6c66-4fb5-8113-e496ce7ac6b0.png">